target/linux/ramips/image/mt76x8.mk

```diff
@@ -621,6 +621,21 @@ define Device/tplink_archer-mr200-v5
 endef
 TARGET_DEVICES += tplink_archer-mr200-v5
 
+define Device/tplink_archer-mr200-v6
+  $(Device/tplink-v2nolzma)
+  IMAGE_SIZE := 15936k
+  DEVICE_MODEL := Archer MR200
+  DEVICE_VARIANT := v6
+  TPLINK_FLASHLAYOUT := 16Mqca
```

Comment:
You must extend a 16Mmtk layout for tplink firmware utils. The qca target has different kernel load and entry addresses. Maybe there are also some endian issues. Ref:
https://github.com/openwrt/firmware-utils/blob/f35781fcd2c3c914247fdb941dc6682af5b98982/src/mktplinkfw2.c#L159

Reply:
I added a 16Mmtk layout but it still didnt boot with sysupgrade and tplink header.  What kind of endian issues? do you mean header related?

Reply:
Header was fine, I needed to set tplink-v2-image instead of tplink-v2-header.